base/linalg/bunchkaufman.jl

```diff
@@ -47,6 +53,22 @@ function bkfact!{T<:BlasComplex}(A::StridedMatrix{T}, uplo::Symbol=:U,
     end
     BunchKaufman(LD, ipiv, char_uplo(uplo), symmetric, rook, info)
 end
+
+"""
+    bkfact(A, uplo::Symbol=:U, symmetric::Bool=issymmetric(A), rook::Bool=false) -> BunchKaufman
+
+Compute the Bunch-Kaufman [^Bunch1977] factorization of a real symmetric or complex Hermitian
```

Comment:
This is wrong so it should be fixed at some point. Might make sense to do after you have moved the doc string though.